traad/app.py

```diff
@@ -181,10 +181,29 @@ def get_imports_view(context):
 @standard_refactoring
 def rename_view(context):
     args = bottle.request.json
+
+    log.info("`rename_view` called: {}".format(args))
+
+    # TODO 1: Modify "emacs-traad" to use these parameters.
+    if 'docs' in args:
+        docs = bool(args['docs'])
+    else:
+        docs = False
```

Comment:
I think a better way of expressing this would be `docs = bool(args.get('docs', False))`. Same for the other checks you make below.

Reply:
Sounds good. I'll swap them around.